dotCMS/src/main/java/com/dotcms/publisher/business/PublisherQueueJob.java

```diff
@@ -207,8 +209,6 @@ public void execute(JobExecutionContext jobExecutionContext) throws JobExecution
 				HibernateUtil.closeSession();
```

Comment:
shouldn't be removed this one HibernateUtil.closeSession(); too?

Reply:
yeah same thing, removed the entire `finally`